app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,15 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str, bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def get_rating(self) -> int:
+        return 3 * self._bow_level
+
+    def player_info(self) -> str:
+        return (f"Elf ranger {self.nickname}. "
```

Comment:
The code uses two different styles of quotes: single quotes in the import statement and double quotes in the f-string. According to the checklist, double quotes are preferable, so it's better to use double quotes consistently throughout the code.